src/NUnitFramework/framework/Internal/SandboxedThreadState.cs

```diff
@@ -34,6 +34,9 @@ internal sealed class SandboxedThreadState
     {
         public CultureInfo Culture { get; }
         public CultureInfo UICulture { get; }
+
+        /// <summary>Thread principal</summary>
+        /// <value>This will be null on platforms that don't support <see cref="Thread.CurrentPrincipal"/></value>
```

Comment:
Since `<value>` is not shown in intellisense, please use `<summary>` for that and add a period at the end.